app/custom_auth.py

```diff
@@ -0,0 +1,15 @@
+from flask_basicauth import BasicAuth
+from flask import jsonify, request
+
+class CustomBasicAuth(BasicAuth):
+    """
+        Description: 
+        Override BasicAuth to permit anonymous healthcheck at /_status?simple=true
+    """
+    def challenge(self):
+        if "/_status" in request.url:
+            if request.args.get('elb', None) or request.args.get('simple', None):
```

Comment:
Would it make sense to allow all requests to the status check endpoint? I'm curious when we would expect to see the ELB param

Reply:
The elb param is vestigial to the UK deployment, we can probably remove it. Right now this is a bit of a hack to allow the healthcheck to pass by hardcoding the response. Removing the auth and passing the actual response from the route is a pain because BasicAuth does not have something like @does_not_require_auth. So we're forced to either turn auth on for all routes using the BASIC_AUTH_FORCE global var, or go through every route (except status) and add an @requires_auth decorator